threescale/src/lib.rs

```diff
@@ -1,4 +1,5 @@
 #![deny(clippy::all, clippy::cargo)]
+#![allow(clippy::multiple_crate_versions)]
```

Comment:
@unleashed Had to add this because there was conflict between dependencies of `threescalers` and `rust-url` and clippy doesn't pass. 
https://rust-lang.github.io/rust-clippy/master/index.html#multiple_crate_versions

Reply:
You should not need this if you patch the dependency in the main Cargo.toml for the workspace with something like:

```toml
[patch.crates-io]
percent-encoding = { git = "https://github.com/3scale-rs/rust-url", branch = "3scale" }
```

Reply:
Okay great !! It worked.